students/hyeongtaek/westagram/users/views.py

```diff
@@ -28,10 +30,12 @@ def post(self,request):
             if not re.match(REGEX_PASSWORD , password):
                 return JsonResponse({"message" : "INVALID_PASSWORD"}, status = 400)
 
+            hashed_password = bcrypt.hashpw(data['password'].encode('UTF-8'), bcrypt.gensalt())
+
             User.objects.create(
                 name          = name,
                 email         = email,
-                password      = password,
+                password      = hashed_password.decode('utf-8'),
```

Comment:
로직을 위에서 다 처리하신 후에 create 해주는 것이 바람직해 보입니다!
```suggestion
                password      = hashed_password,
```